providers/hns/hns_roce_u.h

```diff
@@ -183,7 +183,7 @@ struct hns_roce_srq {
 	unsigned int			srqn;
 	int				max;
 	unsigned int			max_gs;
-	int				wqe_shift;
+	unsigned int			wqe_shift;
```

Comment:
The same "int wqe_shift" in struct hns_roce_wq, it should be updated too.

Reply:
Thanks, we will fix it